config/schemas/config.yaml

```
@@ -5855,6 +5855,18 @@ properties:
             $ref: '#/$defs/kubernetesNodeSelector'
           affinity:
             $ref: '#/$defs/io.k8s.api.core.v1.Affinity'
+            cookieTtl:
+              title: OpenSearch Dashboards Cookie TTL
+              description: Time-to-live for the session cookie in milliseconds. Overrides OpenSearch Dashboards internal default if set.
+              type: integer
+            sessionTtl:
+              title: OpenSearch Dashboards Session TTL
+              description: Time-to-live for the session itself in milliseconds. Overrides OpenSearch Dashboards internal default if set.
+              type: integer
+            sessionKeepalive:
+              title: OpenSearch Dashboards Session Keepalive
+              description: Whether the session TTL should be extended upon user activity. Overrides OpenSearch Dashboards internal default if set.
+              type: boolean
```

```suggestion
          cookieTtl:
            title: OpenSearch Dashboards Cookie TTL
            description: Time-to-live for the session cookie in milliseconds. Overrides OpenSearch Dashboards internal default if set.
            type: integer
          sessionTtl:
            title: OpenSearch Dashboards Session TTL
            description: Time-to-live for the session itself in milliseconds. Overrides OpenSearch Dashboards internal default if set.
            type: integer
          sessionKeepalive:
            title: OpenSearch Dashboards Session Keepalive
            description: Whether the session TTL should be extended upon user activity. Overrides OpenSearch Dashboards internal default if set.
            type: boolean
```
Looks like this ended up inside the affinity key, right?

---

Yes, added the commit @lunkan93